### DISCUSSION TITLE
RLiveObject, I can't set my fields to null.

### DISCUSSION THREAD

**Discussion by yoursalvation** [redisson/redisson#5833] [Q&A] [answered]
Hello,
I've been using RLiveObjects and for some reason I cannot set the fields of my RLiveObject to null. I've read the docs and couldn't find anything about this / in the documentation.

The setter of the field has nothing special in it, no returns, no nothing. The value will be set to what I put as a parameter, altough it doesn't.

        myObject.setTest("a");
        System.out.println(myObject.getTest());

//prints "a"

        myObject.setTest("b");
        System.out.println(myObject.getTest());

// prints "b"

        myObject.setTest(null);
        System.out.println(myObject.getTest());

// prints "b"

If there is more information that I need to provide, please tell me.

**Comment by mrniko** [comment]
try to set `readMode = MASTER`

**Reply by yoursalvation** [reply]
Hey, I managed to get a cluster running, I set the readMode to Master but this didn't resolve the issue. Is there anything else I can try?

**Reply by mrniko** [reply]
Can you share any code sample to reproduce it?

**Reply by yoursalvation** [reply]
```
public static void main(String[] args) {
        RedissonClient redis = Redisson.create();
        LiveObject liveObject = redis.getLiveObjectService().merge(new LiveObject());

        liveObject.setTest("a");
        System.out.println(liveObject.getTest());
        //prints a
        liveObject.setTest("b");
        System.out.println(liveObject.getTest());
        //prints b
        liveObject.setTest(null);
        System.out.println(liveObject.getTest());
        //prints b
    }
```

```
@REntity
public class LiveObject {
    @RId
    private final UUID uuid;
    private String test;

    public LiveObject() {
        this.uuid = UUID.randomUUID();
    }

    public void setTest(String test) {
        this.test = test;
    }

    public String getTest() {
        return test;
    }
}
```

This replicates the problem.

**Reply by mrniko** [reply]
Thanks for the provided test. Can you create an issue?

**Reply by yoursalvation** [reply]
Sure, so that means it's a bug?

**Reply by mrniko** [reply, marked as answer]
yes
